lib/tapioca/sorbet_ext/generic_name_patch.rb

```diff
@@ -20,31 +20,29 @@ def [](*types)
         Tapioca::Runtime::GenericTypeRegistry.register_type(constant, types)
       end
 
-      def type_member(variance = :invariant, fixed: nil, lower: T.untyped, upper: BasicObject)
+      def type_member(variance = :invariant, &blk)
```

Comment:
Should we instead support both syntaxes at the same time so that devs can update tapioca without migrating their codebases?

Reply:
Yes, I would really like this to be backwards compatible, especially since I would like to backport this to the 0.7 series as well, since it is blocking people from updating their Sorbet versions.